nft-packs/src/state/proving_process.rs

```diff
@@ -28,7 +28,7 @@ pub struct ProvingProcess {
     ///
     pub cards_redeemed: u32,
     /// BTreeMap with cards to redeem and statuses if it's already redeemed
-    pub cards_to_redeem: BTreeMap<u32, bool>,
+    pub cards_to_redeem: BTreeMap<u32, u32>,
```

Comment:
Change LEN of account as well

Reply:
Is structure hold 5 bytes before refactoring?: u32 - 4bytes, bool - 1byte

Reply:
yep, for one element it was 4
now it should be 8 for one element